src/components/inputs/Select/multi.js

```diff
@@ -45,11 +45,13 @@ const MultiSelect = ({
     const { formatMessage } = useSafeIntl();
     const classes = useStyles();
     const shiftKeyIsDown = useKeyPressListener('Shift');
+    //  Handle numeric 0 as value
+    const hasValue = Boolean(value)|| value === 0
```

Comment:
there is a missing white space here no ?

Reply:
yeah, my linter seems to be on strike for this repo